daisy_workflows/examples/README.md

```diff
@@ -0,0 +1,41 @@
+# Daisy Examples
+This is a collection of Daisy workflow examples. To run any of these examples,
+you need to replace some values in the workflow configs. In particular, you
+must at least change the workflow `Project`, `GCSPath`, and `OAuthPath` fields
```

Comment:
You can't override these with the -variables flag. I think you mean the -project, -gcs_path, and -oauth flags in daisy. Maybe even drop the OAuthPath fields from the workflows as that one can be taken from the environment